src/fungible_token.rs

```diff
@@ -27,6 +27,17 @@ pub struct FungibleToken {
     pub account_storage_usage: StorageUsage,
 }
 
+#[derive(Default, BorshDeserialize, BorshSerialize, Clone)]
```

Comment:
Instead of deriving `Default`, I think it might be nice to have a constructor that returns the values for ETH on Aurora (since that is the primary use case for this struct).

Reply:
What did you mean? How is it related to Metadata?

Reply:
Having empty metadata is never useful.

The only time we use this struct is to describe the FT which is contained in the Aurora Engine contract, that is the Aurora ETH. So it would be useful for us to have a constant like `FungibleTokenMetadata::ETH` which is used in places where we currently have calls to `default()`. This could also be useful in the state migration.

Reply:
Why it is empty??? We initialized that.
Default - only for tests. We can implement something different for tests, if we need.

Reply:
I understand right now that `Default` is only used in tests.

What I am saying is that having this `Default` implementation is misleading because it implies that empty metadata is a reasonable value, but it is not.

Instead we could have a constant which contains the metadata that is useful for our (Aurora) use case.

Reply:
Good point. I'll remove it. And put some default impl in tests.